src/utils/images/Images.js

```diff
@@ -22,17 +22,22 @@ function makeGDriveLinkUsable(s)
 // End name with a comma (Place links within quotes)
 // First 4 names will be displayed
 const ProjectImg = [
-    'https://drive.google.com/file/d/135jImRhC8MJz-A2UiP_Kj7yjeTjU2Rba/view?usp=sharing',
-    'https://drive.google.com/file/d/1DKDKMaLfrMFLyRolcnHMu3A8RSJoLm84/view?usp=sharing',
-    'https://drive.google.com/file/d/1SJ2NJ8NVspmSCV70vxmPRz83W3LAqyEu/view?usp=sharing',
-    'https://drive.google.com/file/d/1Rhd4nX2gKiI4Ou1j_EtqiTVe67IyG09f/view?usp=sharing',
+    {
+        src: 'https://drive.google.com/file/d/1ven0z8524cKmY3cnJLQyjhXuTVCsYD2o/view?usp=sharing', 
+        id: "webapp"
+    },
+    
+    {
+        src: 'https://drive.google.com/file/d/1DsND2x20HvDsdY7Ve4PTfyGT-XSYhlPD/view?usp=sharing', 
+        id: "stock"
+    },
 ];
```

Comment:
For now this is file but create a JSON file in static folder and import from there.

Reply:
Ok sir.. actually we will use something from backend for images instead of google drive.. This one is temporary. I will make the necessary changes after talking with backend workers